it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
-12591=Non è stato possibile inviare il messaggio in quanto la connessione al server SMTP %S è stata persa nel mezzo della transazione. Riprovare oppure contattare l'amministratore del server.
+12591=Non è stato possibile inviare il messaggio poiché la connessione al server SMTP %S è stata persa durante la transazione. Riprova oppure contatta l'amministratore del server.
 
 ## @name NS_ERROR_SMTP_SEND_FAILED_TIMEOUT
-12592=Non è stato possibile inviare il messaggio in quanto la connessione al server SMTP %S è andata fuori tempo massimo. Riprovare oppure contattare l'amministratore del server.
+12592=Non è stato possibile inviare il messaggio poiché la connessione al server SMTP %S è andata fuori tempo massimo. Riprova oppure contatta l'amministratore del server.
 
 ## @name NS_ERROR_SMTP_SEND_FAILED_UNKNOWN_REASON
-12593=Non è stato possibile inviare il messaggio attraverso il server SMTP %S per una ragione sconosciuta. Verificare le impostazioni SMTP e riprovare oppure contattare l'amministratore del server.
+12593=Non è stato possibile inviare il messaggio attraverso il server SMTP %S per motivi sconosciuti. Verifica le impostazioni SMTP e riprova oppure contatta l'amministratore del server.
 
 ## @name NS_MSG_ERROR_DOING_DRAFTS
-13000=There was an error copying the message to the Drafts folder. Retry?
+13000=Si è verificato un errore durante la copia del messaggio nella cartella Bozze. Riprovare?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 # LOCALIZATION NOTE (12594): $S is server hostname
-12594=There was an error copying the message to the Drafts folder. Retry?
+12594=Si è verificato un errore durante la copia del messaggio nella cartella Bozze. Riprovare?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 # LOCALIZATION NOTE (12595): $S is server hostname
-12595=The SMTP server %S does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+12595=Sembra che il server SMTP %S non supporti le password crittografate. Se stai impostando questo account, prova a impostare 'Password normale' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'. Se prima funzionava ma improvvisamente ora non funziona, contatta il tuo amministratore o fornitore e-mail.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 # LOCALIZATION NOTE (12596): $S is server hostname
-12596=The SMTP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+12596=Il server SMTP %S non consente password con testo semplice. Prova a cambiare in 'Password crittografate' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+12596=Il server SMTP %S non consente password con testo semplice. Prova ad utilizzare 'Password crittografate' come 'Metodo di autenticazione' nelle 'Impostazioni account | Impostazioni server'.